apps/src/netsim/NetSimTable.js

```diff
@@ -217,8 +210,7 @@ NetSimTable.prototype.subscribe = function () {
  * local channel. Also clears the saved callback.
  */
 NetSimTable.prototype.unsubscribe = function () {
-  this.channel_.unsubscribe(this.tableName_, this.channelCallback_);
-  this.channelCallback = undefined;
+  this.channel_.unsubscribe(this.tableName_);
```

Comment:
Could you update the comments to reflect the minor functionality change here? Also, given that we're no longer specifying which callback we're unsubscribing, it would be nice to make explicit that we're unsubscribing everything